bitnami/nginx/ci/values-with-ingress-metrics-and-serverblock.yaml

```diff
@@ -24,3 +24,8 @@ metrics:
   ## Kubeval doesn't recognise ServiceMonitor as a valid K8s object
   # serviceMonitor:
   #   enabled: true
+
+pdb:
+  create: true
+  minAvailable: 3
+  maxUnavailable: 5
```

Comment:
Why are you adding it to this file?

Reply:
Because I want to have the PDB included in the tests. I thought it would be a good idea to test even this tiny code sniped?!

Reply:
Ok, I agree it is a good idea to add it to the tests. Thanks!